package.json

```diff
@@ -106,7 +106,9 @@
         "typescript": "^3.9.6",
         "webdriverio": "^6.4.6"
     },
-    "dependencies": {},
+    "dependencies": {
+        "intersection-observer": "^0.11.0"
```

Comment:
👀 Open question about how we include these kinds of polyfills. For now I've included it here and imported it directly into `oisc-warnings.js`. This this a reasonable approach?

Reply:
When I run a build only the `require` statement for this polyfill is included, so definitely open to suggestions on how best to include this here (or should it be a requirement for the consumer?) @baseonmars @davidsauntson